packages/directlinespeech/package.json

```diff
@@ -60,9 +60,9 @@
     "event-target-shim": "^5.0.1",
     "event-target-shim-es5": "1.0.1",
     "math-random": "^1.0.4",
-    "microsoft-cognitiveservices-speech-sdk": "1.9.1",
+    "microsoft-cognitiveservices-speech-sdk": "1.10.1",
     "p-defer": "^3.0.0",
     "p-defer-es5": "1.0.1",
-    "web-speech-cognitive-services": "6.3.0"
+    "web-speech-cognitive-services": "7.0.0-master.da7babf"
```

Comment:
needs to be reviewed.

Reply:
This is the compare, I will include it in top conversation for visibility.

https://github.com/compulim/web-speech-cognitive-services/compare/v6.3.0..da7babf

Reply:
@beyackle if you have the time, would you mind looking at the code above? This is a package we use in Web Chat, and those changes also need to be reviewed.

Reply:
This looks like just a version bump for a few packages, and seems all right to me.